### PR TITLE
Travis webhook for builds

### PR DESCRIPTION
**Pull request by drewvolz**
Adds a webhook for travis builds.

### CODE REVIEW

File: .travis.yml
Comment (hawkrives) [inline .travis.yml]: Um. What is this.

**Review by hawkrives — DISMISSED** [review]

File: .travis.yml
Comment (rye) [reply .travis.yml]: I'd guess a variable called `TRAVIS_WEBHOOK`.

Probably defined in the Travis CI web configuration.

Comment (drewvolz) [reply .travis.yml]: This is most definitely problematic. Is there a way we can make a variable from the URL without revealing it?

> Note: These webhooks are executed at the end of a build, and not by individual jobs (see builds vs jobs). This means that environment variables from the build are not available in this section.

Comment (hawkrives) [reply .travis.yml]: Why don't we reveal it? We can destroy and recreate the webhook in Discord very easily

Comment (drewvolz) [reply .travis.yml]: Perfect. I was concerned about privacy but I suppose there's no reason for that.

**Comment by drewvolz** [comment]
~Fixed webhook to not be a url~
Fixed webhook to not be a variable

**Review by rye — APPROVED** [review]
If we're sure there's no way to get this webhook out into a variable, LGTM.

**Comment by hawkrives** [comment]
I mean, it's still not working, since there's nothing showing up in Circle, so it could go back into a variable.

**Comment by drewvolz** [comment]
The docs mentioned it not being able to see env vars at this point.

**Comment by hawkrives** [comment]
Well. I'd hold off on this still; we should be able to get it to notify Discord from a PR.

**Comment by drewvolz** [comment]
Close it?

**Comment by hawkrives** [comment]
If you like

**Pull request closed without merging**
